requirements.txt

```diff
@@ -24,3 +24,6 @@ transformers
 typing_extensions==4.7.1
 validators
 xlsxwriter
+timm
+einops
+flash_attn
```

Comment:
Should not include those model specific dependencies (a large number of MLLMs do not depend on those libs), which will make it more difficult to install the codebase.